src/normalizer.ts

```diff
@@ -145,6 +145,30 @@ rules.set('Transform const to singleton enum', schema => {
   }
 })
 
+rules.set('Transform nullable -> anyOf', schema => {
+  if (schema.nullable === true) {
+    delete schema.nullable
+
+    const copiedSchema = {...schema}
+
+    // This stuff should not be in `anyOf`.
+    delete copiedSchema.id
+    delete copiedSchema.nullable
+
+    // This stuff will be in `anyOf` instead.
+    delete schema.format
+    delete schema.type
+
+    schema.anyOf = [
+      copiedSchema,
+      // @ts-expect-error
```

Comment:
Can you rm this?

Reply:
I get a TS error without it:

```
Type '{ type: "null"; }' is missing the following properties from type 'LinkedJSONSchema': additionalProperties, [Parent]ts(2739)
```

Reply:
Cool! Can you find a way to fix the error? eg. to add an empty `additionalProperties`, and to add a `Parent` field?